qiskit_experiments/library/calibration/half_angle_cal.py

```diff
@@ -38,10 +38,12 @@ def __init__(
         calibrations: Calibrations,
         backend: Optional[Backend] = None,
         schedule_name: str = "sx",
-        cal_parameter_name: Optional[str] = "amp",
+        cal_parameter_name: Optional[str] = "angle",
         auto_update: bool = True,
     ):
-        """see class :class:`HalfAngle` for details.
+        """see class :class:`HalfAngle` for details. The experiment updates the
```

Comment:
I prefer having simple one line description because it appears in the autosummary. Something like `see class ...` can be removed because #1077 automatically inserts reference to parent classes.